packages/dds/tree/src/simple-tree/core/treeNodeSchema.ts

```diff
@@ -121,6 +121,16 @@ export interface TreeNodeSchemaCore<
 	out ImplicitlyConstructable extends boolean,
 	out Info = unknown,
 > {
+	/**
+	 * Unique (withing a document's schema) identifier used to associate nodes with their schema.
```

Comment:
```suggestion
	 * Unique (within a document's schema) identifier used to associate nodes with their schema.
```

Reply:
Nit: Might also be a bit easier to read if you move the parens block after "identifier".